sim/paladin/paladin.go

```diff
@@ -77,18 +81,27 @@ func (paladin *Paladin) Initialize() {
 	paladin.registerHammerOfWrath()
 	paladin.registerHolyWrath()
 	paladin.registerAvengingWrath()
+	paladin.registerAuraMastery()
+
+	if paladin.primaryPaladinAura == proto.PaladinAura_SanctityAura {
+		paladin.currentPaladinAura = core.SanctityAuraAura(paladin.GetCharacter())
+	}
```

Comment:
I think this shouldn't be needed anymore after adding the `ResetCurrentPaladinAura` bit below